test/api/publish.test.js

```diff
@@ -207,7 +207,7 @@ describe('API versioning', () => {
   });
 
   it('returns an error when an invalid `accept` header is specified', async () => {
-    const req = getPreconfiguredRequest().accept('INVALID_VALUE');
+    const req = getPreconfiguredRequest().accept('invalid/mime-type');
```

Comment:
Without this, I get a `500` error instead of the expected `406`. I haven't been able to find the root cause of why I needed this, but I've seen that `supertest` updated a lot of dependencies (including `express` and `nock`) and my guess is that one of these now checks for the _syntax_ of the `Accept` header value before handing it over to our code.

(note: `supertest` runs our code inside `express` -- which isn't exactly the same as our runtime, because we use `koa` -- not perfect but well...)